rest-service/manager_rest/rest/resources_v3_1/labels.py

```diff
@@ -0,0 +1,17 @@
+from manager_rest.security import SecuredResource
+from manager_rest.security.authorization import authorize
+from manager_rest.storage import models, get_storage_manager
+
+
+class DeploymentsLabels(SecuredResource):
+
+    @authorize('labels_keys_list')
+    def get(self):
+        """Get all deployments labels' keys"""
+        sm = get_storage_manager()
+        raw_labels_list = sm.list(models.DeploymentLabel,
+                                  include=['key'],
+                                  get_all_results=True)
+        keys_list = list(set(label.key for label in raw_labels_list))
+        result = {'metadata': {}, 'items': keys_list}
```

Comment:
It might be easier just to return `{'keys': keys_list}`, but I saw this example - https://github.com/cloudify-cosmo/cloudify-manager/blob/309e4958e64ca69e76ef7bae13b255b70a9fe958/rest-service/manager_rest/rest/resources_v3_1/manager_config.py#L31 and figured this is the right way.